### ISSUE TITLE
Question about the Meaning of Accuracy in the Table Results

### ISSUE THREAD

**Issue by EveDong** [YU1ut/MixMatch-pytorch#14]
Does the the accuracy in the table Results refer to "Best acc" or "Mean acc" in the code?

**Comment by xiaopingzeng** [comment]
'Best acc' means the best performance in validation dataset， 'Mean acc' means the mean performance of the last 20 epoch test dataset.

**Comment by EveDong** [comment]
I mean is the accuracy showed in the result table the best accuracy or the mean accuracy?

**Comment by nazim1021** [comment]
That would be the mean accuracy

**Comment by guixianjin** [comment]
I'm confusing with 'Best accu'.  Assuming we have a validation set and a separate test data set, does 'Best accu'  not mean the accuracy in test data set of the epoch where the accuracy in validation set is best? It may be lower than the potential best accuracy in test data set if the epoch selected by validation set is not so good. Am I wrong?  @xiaopingzeng @EveDong
